io/post_fv3.F90

```diff
@@ -799,9 +821,9 @@ subroutine set_postvars_fv3(wrt_int_state,grid_id,mype,mpicomp)
 !$omp& shared(smstav,sfcevp,acsnow,acsnom,qz0,uz0,vz0,maxrhshltr,minrhshltr)
       do j=jsta_2l,jend_2u
         do i=ista_2l,iend_2u
-          smstav(i,j) = spval
+          !smstav(i,j) = spval
           sfcevp(i,j) = spval
-          acsnow(i,j) = spval
+          !acsnow(i,j) = spval
```

Comment:
Remove these two lines if they are note needed anymore. And two variables from omp directive

Reply:
Done!